next.config.js

```diff
@@ -1,71 +1,26 @@
-const path = require('path')
-
-// TODO: we need to check why this file is not parsing by .eslintrc.json
-// eslint-disable-next-line import/no-extraneous-dependencies
 const webpack = require('webpack')
-const glob = require('glob')
 const withPlugins = require('next-compose-plugins')
-const withCSS = require('@zeit/next-css')
-const withSourceMaps = require('@zeit/next-source-maps')
+const css = require('@zeit/next-css')
+const sourceMaps = require('@zeit/next-source-maps')
 const optimizedImages = require('next-optimized-images')
-const withOffline = require('next-offline')
+const fonts = require('next-fonts')
```

Comment:
It's necessary for init project?

Reply:
When we need to work with icon fonts its useful.